dxcint/dxcint/Terraform.py

```diff
@@ -98,33 +98,40 @@ def _make_prerequisites(self, language: str) -> str:
 
     @async_retry()
     def _build_asset(self, language: str) -> str:
+
+        def find_asset_obj(self: Terraform, asset_name: str, zero_ok: bool) -> str:
+            return dxpy.search.find_one_data_object(
+                classname="record",
+                project=self._context.project_id,
+                name=asset_name,
+                folder=self._context.platform_build_dir,
+                more_ok=False,
+                zero_ok=zero_ok
+            )
+
         asset_name = f"dx{language}rt"
+        # skip building, if asset is already present
+        asset_obj = find_asset_obj(self, asset_name, zero_ok=True)
+        if asset_obj is not None:
+            return asset_obj.get("id")
```

Comment:
Nice optimization - but we do wanna rebuild assets for every test run, because we are testing the code changes. And the assets represent those executors of the workflows that are the part of dxCompiler. If you rely on the previous build - you won't be able to test the code changes.
Of note - previous testing script did have some options to skip rebuilding which IMHO made UX more complex and I never used them. Please revert